pkgs/by-name/re/restate/package.nix

```diff
@@ -0,0 +1,125 @@
+{
+  lib,
+  testers,
+  stdenv,
+  rustPlatform,
+  fetchFromGitHub,
+  pkg-config,
+  openssl,
+  perl,
+  rocksdb,
+  protobuf,
+  cmake,
+  restate,
+}:
+let
+  # Have to link against this fork until this PR is merged
```

Comment:
You can put the diff the PR represents into the folder with the `package.nix` and apply it to the source, then pull from the original repo

Reply:
On testing I have realized that it needs more than a single PR I have mentioned to build. A diff against their fork generates 12 patches, so I have removed the nixpkgs input altogether, now `rustPlatform.buildRustPackage` builds it instead. I think, that a benefit of this approach, is that rocksdb is linked statically then.